aws_db_instance.tf

```diff
@@ -3,47 +3,53 @@
 #}
 
 resource "aws_db_instance" "secureInstance" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure the RDS instance follows standard configuration**
 
**Vulnerability**
The RDS Instance doesn’t comply with standard configuration
 
**Remediation**
Ensure the following values are set as per the company standard - DB Instance type: db.m5.xlarge, Allocated Storage: 200 G, Max Allocated storage: 500 G, Performance Insights Enabled: True, Enhanced monitoring: True

source: accurics-